src/ontology/uberon-edit.obo

```diff
@@ -222238,6 +222238,15 @@ synonym: "fifth molar tooth" EXACT []
 is_a: UBERON:0003655 ! molar tooth
 relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
 
+[Term]
+id: UBERON:7500128
+name: temporal crest
+def: "A dorsally concave, laterally-directed crest that terminates on the caudal-dorsal process of a zygomatic bone." [DOI:10.5455/ijlr.20170908094210]
```

Comment:
Considering UBERON:0004661 'mandible temporal crest'... 
If this new term is not a parent class to that, consider relabelling so that this term is differentiated from 'mandible temporal crest'.

The text definition references the zygomatic bone while the logical definition invokes the temporal bone. 
See these references, although they may be specific to humans:

https://academic.oup.com/asj/article/29/2/158/270420?login=true
https://www.imaios.com/en/e-anatomy/anatomical-structure/temporal-crest-1536898724

 If the meaning of this new term is specific to a particular taxon, consider adding taxon constraints.

Reply:
how about cranial temporal crest since the cranium excludes the mandible?

We will add the taxon constraint for Suidae too.

Reply:
@megbalk, labelling as 'cranial temporal crest' with exact synonym "temporal crest", or vice versa, seems reasonable.

If this structure is the same as [this](https://www.imaios.com/en/e-anatomy/anatomical-structure/temporal-crest-1536898724), consider adding the Latin term as a synonym.

After a cursory search, it does look like the structure in the human reference is also called a temporal line.

Reply:
The cranial temporal crest is bounded by the superior and inferior temporal lines, but is not the same as the temporal line. It is also not the same as the structure you linked, which is on the mandible not the temporal bone. We added clarification on the position of the cranial temporal crest. @helenamachado

I hesitate to make an exact synonym with "temporal crest" since there seems to be two different places it could be (on the mandible or on the temporal bone). Does a broad synonym to "temporal bone" make more sense?

Reply:
Hi, @megbalk,

> The cranial temporal crest is bounded by the superior and inferior temporal lines, but is not the same as the temporal line.

Consider adding a dbxref to support this assertion in the text def.